src/OpenTelemetry.Instrumentation.MySqlData/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 1.0.0-beta.3
```

Comment:
I'd suggest to put the date as well. Example : https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

@utpilla Could we update the docs to say release request should put the date as well?

Reply:
This could be done in a follow-up PR.